src/components/MovieInfo.tsx

```diff
@@ -30,6 +34,7 @@ const MovieInfo: React.FC<MovieInfoProps> = ({ data }) => {
                 <div className="card-body">
                   <div className="card-title lead text-uppercase">{Title}</div>
```

Comment:
Update this part with the below suggestions, please:
`<div className="card-title lead text-uppercase h4 fs-6">{Title}</div>`